<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>708</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>708</width>
    <height>500</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Real-CUGAN - Iceshell</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_upscale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="currentText">
     <string>1</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_selector_out">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Using same folder as input</string>
    </property>
    <property name="text">
     <string>Same Folder as Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Upscale the image by this factor. Set to 2 for best result</string>
    </property>
    <property name="text">
     <string>Upscale</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_denoise">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Remove noise in image. Set to -1 for no denoise</string>
    </property>
    <property name="text">
     <string>Denoise</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_outext">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Same</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>jpg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>webp</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Output image has same file type as input image if no file type is selected</string>
    </property>
    <property name="text">
     <string>Out Ext</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_outdir">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Same</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Select...</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Output image to same folder as the input if no folder is selected</string>
    </property>
    <property name="text">
     <string>Out Dir</string>
    </property>
   </widget>
   <widget class="DdLabel" name="label_file_input">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>711</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>res/ui_bg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Progress of conversion</string>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_about">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <zorder>label_file_input</zorder>
   <zorder>comboBox_upscale</zorder>
   <zorder>pushButton_selector_out</zorder>
   <zorder>label</zorder>
   <zorder>comboBox_denoise</zorder>
   <zorder>label_2</zorder>
   <zorder>comboBox_outext</zorder>
   <zorder>label_3</zorder>
   <zorder>comboBox_outdir</zorder>
   <zorder>label_4</zorder>
   <zorder>progressBar</zorder>
   <zorder>pushButton_about</zorder>
  </widget>
  <action name="action1x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1x</string>
   </property>
  </action>
  <action name="action2x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2x</string>
   </property>
  </action>
  <action name="action3x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3x</string>
   </property>
  </action>
  <action name="action4x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4x</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DdLabel</class>
   <extends>QLabel</extends>
   <header>ddlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="bg.qrc"/>
 </resources>
 <connections/>
</ui>
